tests/models/test_dagbag.py

```diff
@@ -689,40 +691,57 @@ def test_sync_to_db_is_retried(self, mock_bulk_write_to_db, mock_s10n_write_dag,
 
     @patch("airflow.models.dagbag.settings.MIN_SERIALIZED_DAG_UPDATE_INTERVAL", 5)
     @patch("airflow.www.security.ApplessAirflowSecurityManager")
-    def test_sync_to_db_handles_dag_specific_permissions(self, mock_security_manager):
+    @freeze_time(tz.datetime(2020, 1, 5, 0, 0, 0), as_kwarg="frozen_time")
+    def test_sync_to_db_handles_dag_specific_permissions_fixed(self, mock_security_manager, frozen_time):
```

Comment:
```suggestion
    def test_sync_to_db_handles_dag_specific_permissions(self, mock_security_manager, frozen_time):
```

Unless this was intentional :)